jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -245,6 +253,23 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Store the provided AWS virtual private cloud value in state.
+      In the case the value is set, also set whether to force Juju to use the
+      given value, even when it fails the minimum validation criteria.
+
+      @method _setVPCId
+      @param {String} value The VPC identifier.
+      @param {Boolean} force Whether to force the value. Ignored if !value.
+    */
+    _setVPCId: function(value, force) {
+      if (!value) {
+        value = null;
```

Comment:
I wonder if we shouldn't define defaults for these values then this would become something like...
```JavaScript
...
value = this.DEFAULTS.vpcId;
...
```